assets/mentorRequest.html

```diff
@@ -1,154 +1,72 @@
-
-<!-- ------------------------to create the inline css in the HTML: https://foundation.zurb.com/emails/inliner-v2.html --------------------- -->
-
-<!-- ------------------------the original html --------------------- -->
-<!-- <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
 
-
-  <link href="https://fonts.googleapis.com/css?family=Poppins:400,600" rel="stylesheet">
-
+    tbody {
+      border-color: #fff0;
+    }
+  </style>
 </head>
 
 <body>
-  <div id="main">
-    <table>
-      <tbody>
-        <tr id='logo'>
-          <td class='textalign-center'>
-            <img src='https://beta.upframe.io/emailLogo.png' alt='Upframe' title='Upframe' style='display: block' width='203' height='41'></img>
-          </td>    
-        </tr>
-
-        <tr>
-          <td>
-            <p>Hi MENTOR,<br><br> We're reaching out to let you know  <a target='_blank noreferrer' href="mailto:EMAIL">USER</a> invited
-            you to a <a target='_blank noreferrer' href='LOCATION'>MEETUPTYPE</a> on DATE at TIME(TZ time)<br>
-
-            </p>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <p><strong>USER left you a message:</strong></p>
-            <p class= 'textarea'MESSAGE</p>
-          </td>
-        </tr>
-        
-        <tr class='btn-column'>
-          <td class='btn-wrapper'>
-            <a target='_blank noreferrer' href='https://connect.upframe.io/meetup/confirm/MID'>
-              <div class='btn btn-cancel'>cancel</div>
-            </a>
-            <a target='_blank noreferrer' href='https://connect.upframe.io/meetup/confirm/MID'>
-              <div class='btn btn-accept'>Accept</div>
-            </a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </div>
+  <table border="1" cellpadding="0" cellspacing="0" width="600" align="center"
+    style="border-color: #c9cfd4; border-style: solid; border-radius: 10px; font-size: 16px;">
+    <tr>
+      <td align="center" valign="top">
+        <table border="0" cellpadding="0" cellspacing="20">
+          <tr>
+            <td align="center">
+              <img src="https://beta.upframe.io/emailLogo.png" alt="Logo"
+                title="Logo" width="80" height="80"
+                style="display: block;"></img>
+            </td>
+          </tr>
+          <tr>
+            <td width="500"
+              style="font-family: 'Poppins', sans-serif; line-height: 26px; letter-spacing: 0.16px;">
```

Comment:
My friend @MathisBullinger I don't think you can use custom fonts on email. Can you take a look into this? I remember back when we tried at least it wasn't working